include/goofit/PdfBase.h

```diff
@@ -120,6 +120,10 @@ class PdfBase {
     void registerFunction(std::string name, const T &function) {
         reflex_name_  = name;
         function_ptr_ = get_device_symbol_address(function);
+        // add the following 211222 to keep track of the correspondence
+        // between function pointer values and their names to make
+        // debugging/tracking the executionn of the code easier.  mds
```

Comment:
No need to add MDS's to everything, just adds clutter, and you can attribute the git commit to @mdsokoloff instead as a coauthor.

Reply:
Mike is already the author of the first commit. In fact, these comments are directly from him, I just did not change it. I will clean these comments in my next push.